provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerController.java

```diff
@@ -48,65 +53,50 @@
 
   private Practitioner.Bundle bundle(
       MultiValueMap<String, String> parameters, int page, int count) {
-    PractitionerWrapper root = search(parameters);
+    Pair<List<PractitionerWrapper>, Integer> root = search(parameters);
     LinkConfig linkConfig =
         LinkConfig.builder()
             .path("Practitioner")
             .queryParams(parameters)
             .page(page)
             .recordsPerPage(count)
-            .totalRecords(1)
+            .totalRecords(root.getRight())
             .build();
     return bundler.bundle(
         BundleContext.of(
             linkConfig,
-            Collections.singletonList(root),
+            root.getLeft(),
             transformer,
             Practitioner.Entry::new,
             Practitioner.Bundle::new));
   }
 
+  /** Builds the PractitionerWrapper and returns it. */
+  private PractitionerWrapper practitionerWrapperBuilder(
```

Comment:
`practitionerWrapper` is probably a better method name here, since the method is returning a wrapper, not a builder.